### PR TITLE
fix(spinner-class): add glyphicon to default spinner class, update css

### LINKED ISSUE #1893: spinner-class should use glyphicon
The issues forum is __NOT__ for support requests. It is for bugs and feature requests only.
Please read https://github.com/angular-ui/ui-select/blob/master/CONTRIBUTING.md and search
existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.

The spinner-class should use the glyphicon class instead of declaring the font on ui-select-refreshing. If you try to replace `glyphicon-reverse` with a font awesome (`fa fa-spinner fa-pulse fa-fw`) does not work because ui-select-refreshing declares the font.

Spinner class should be `glyphicon glyphicon-refresh ui-select-spin`

and ui-select-refreshing should be
```
.ui-select-refreshing {
    position: absolute;
    right: 0;
    padding: 8px 27px;
 }
```

I can do a PR if you like.

Angular: 1.5.8

UI-Select: 0.19.5

Bootstrap/Select2/Selectize CSS (if applicable): Bootstrap 3.2

### PR DESCRIPTION
**Pull request by brianchance**
Fixes #1893.

### CODE REVIEW

**Comment by Jefiozie** [comment]
@brianchance , can you have a look at the build error's?
Thanks

**Comment by brianchance** [comment]
Done.

File: src/common.css [hunk 1]
```diff
@@ -317,17 +317,10 @@ body > .ui-select-bootstrap.open {
 }
 
 /* Spinner */
-.ui-select-refreshing {
+.ui-select-refreshing.glyphicon {
```
Comment (Jefiozie) [inline src/common.css]: Should the `ui-select-refreshing.glyphicon` be as one selector?

**Comment by brianchance** [comment]
I did it this way so you can target font-awesome with .ui-select-refreshing.fa. It needs different positioning.

**Comment by brianchance** [comment]
For font-awesome, I am using...
```
.ui-select-refreshing.fa {
  position: absolute;
  top: 9px;
  right: 27px;
 }
```

**Comment by Jefiozie** [comment]
Merged it thanks.

**Pull request merged**